test/nightly/deploy.test.ts

```diff
@@ -44,42 +41,45 @@ interface IParallelTest {
     callback(): Promise<void>;
 }
 
-suite('Create Web App and deploy', function (this: Mocha.Suite): void {
+suite.only('Create Web App and deploy', function (this: Mocha.Suite): void {
```

Comment:
I figure that this was for testing purposes, but you should remove the `.only` from the suite now.

Reply:
See original note, but I planned to keep this in until I add in the follow-up PR if that's okay.